TASK1/output.php

```diff
@@ -0,0 +1,9 @@
+ <?php
```

Comment:
1/ Indentation in this file is incorrect.
2/ It looks unnecessary to create a seaprate file just for this purpose.